services/StoreService.js

```diff
@@ -1,20 +1,75 @@
 const prisma = require("../lib/prisma");
 
-const findAll = async (params) => {}
+const findAll = async (params) => {
+  const data = await prisma.stores.findMany();
+  return data;
+};
 
-const findOne = async (params) => {}
+const findOne = async (params) => {
+  const data = await prisma.stores.findUnique({
+    where: {
+      id: params,
+    },
+  });
+  if (!data) {
+    throw Error;
```

Comment:
disini harusnya throw error not found

Reply:
jd abis throw new Error("not found") gitu ya guru? atau console log abis trhow?